Gemfile

```diff
@@ -3,6 +3,5 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in make_flaggable.gemspec
 gemspec
 
-gem "database_cleaner", "1.0.1"
```

Comment:
Did you remove this because it caused dependency problems, or was it just never used?